src/text_file_functions.cpp

```diff
@@ -896,6 +896,19 @@ QString AOApplication::get_custom_realization(QString p_char)
   return get_sfx_suffix(get_sounds_path(f_result));
 }
 
+bool AOApplication::get_pos_is_judge(QString p_pos)
+{
+  QStringList positions = read_design_ini("judges", get_background_path("design.ini")).split(",");
+  if (positions.isEmpty())
+      return p_pos == "jud"; //Hardcoded BS only if we have no judges= defined
```

Comment:
Hmm actually this would be better if judges= by itself would mean there's no judge pos (aka the hardcoding BS doesn't come into effect), and having no judges= period would be the sign that we need to fall back to hard-coding.

Reply:
Hmm, but if there's no judge pos, chances would be slim for there to be a position named `jud`.

Reply:
If there's no judge positions defined, we fall back on the old way of determining judge controls - which is to check if we're on pos `jud`.

Reply:
we're not returning `p_pos`, we're returning a boolean